### ISSUE TITLE
Battery status not accurate

### ISSUE THREAD

**Issue by GabVenturato** [numixproject/numix-icon-theme#966]
In Fedora 22 with GNOME3 the battery icon status is not accurate:
it stays at about half full even if the remaining charge is about 20%, only when it goes down above 20% it became orange and under the half.
and also when i recharge my laptop even if it is at 80% the icon is completely full.
I'm the only one with this problem? I can't find anything on it.
thank you

**Comment by Foggalong** [comment]
Do you have this problem with other icon themes or just ours?

**Comment by GabVenturato** [comment]
Great question, I tried only numix because I love it, but I will try some others and I will tell you if the problem is related only to numix or not!

**Comment by wa4557** [comment]
Seems to me that there are icons missing, and if they are not found the system falls back to the full battery icon. No idea which icons though. Are you using a different battery indicator or the standard one?

**Comment by GabVenturato** [comment]
I'm using the numix icon theme, so the indicator is a numix icon. I see that the standard for GNOME has stripes to indicate the power level, I don't know if it could be related to this thing maybe. I will test better with different icon themes in next days and I will refer you what happens.

**Comment by Foggalong** [comment]
@GabVenturato we meant which battery indicator program are you using

**Comment by GabVenturato** [comment]
Oh sorry, yes I use the standard battery indicator.
Il 23/Dic/2015 21:57, "Joshua H. Fogg" <notifications@github.com> ha
scritto:

**Comment by Foggalong** [comment]
Cool, thanks :+1:

**Comment by wa4557** [comment]
Any other GNOME users experiencing this @dirtydancing @bil-elmoussaoui? Sounds like a serious bug I'm wondering why this hasen't come up yet...

**Comment by Foggalong** [comment]
Another wonderful exploration into the shittiness which is the battery indicator icons.

**Comment by bil-elmoussaoui** [comment]
I have never looked at the battery indicator. I think that i have an issue with the battery indicator too.. 
I'm using Gnome 3.18.3 with Fedora 23 and the battery indicator icon does not seem to be accurate. The `battery-full-symbolic` is used even if my battery is at 68%.. 
![screenshot from 2015-12-23 22-53-15](https://cloud.githubusercontent.com/assets/7660997/11985717/fb8aa9e0-a9c7-11e5-94b3-be0cf8b3b274.png)

**Comment by Foggalong** [comment]
Oh shit. This is actually a big problem. Yaaaaaaaaay \O/

**Comment by dirtydancing** [comment]
Well, to me this looks like the normal GNOME battery indicator with its thresholds. Ubuntu is way more differentiated in this regard. For the thresholds, please cf. e.g. this former comment: https://github.com/numixproject/numix-icon-theme/issues/107#issuecomment-53206000. iirc the thresholds for GNOME are at 60%, 30%, 10%. Ubuntu in contrast currently has 10% steps, cf. overview at https://github.com/numixproject/numix-icon-theme/issues/107#issuecomment-98399311.

@GabVenturato Can we have a screenshot of the specific issue? This makes it easier to address it.

As far as I can tell, no Numix issue here, this is a "feature" of the GNOME battery indicator/power manager :)

**Comment by dirtydancing** [comment]
To clarify from the overview above: Ubuntu/Unity is using the percentage-number-icon-names, whereas GNOME is using the (much fewer) status-icon-names `full, good, low, caution, empty`.

**Comment by wa4557** [comment]
But @GabVenturato said that it is working with the Adwaita theme?

**Comment by dirtydancing** [comment]
That is why I asked for screenshots, as I cannot tell what exactly @GabVenturato is referring to :)

**Comment by dirtydancing** [comment]
Here is a screenshot with the Numix icons above and the Adwaita icons below (when discharging):

![battery-status](https://cloud.githubusercontent.com/assets/7164227/11986247/a266872a-a9cd-11e5-8509-263e1c1e9b76.png)

With GNOME, it is expected behaviour to have the `full` icon from 100% down to 60%, and also the other way around when charging (with a charging flash symbol) to have `full-charging` up from 60%.

**Comment by bil-elmoussaoui** [comment]
@dirtydancing Can confirm what are you saying. The `battery-good-symbolic` was used once the battery level was 59% (Login of gnome :P) 
![screenshot from 2015-12-23 23-42-07](https://cloud.githubusercontent.com/assets/7660997/11986350/cc243b92-a9ce-11e5-99d9-ee5697fcdede.png)

**Comment by wa4557** [comment]
As far as I understand icon number 2 and 3 are never shown, but it jumps from 1 to 4

**Comment by GabVenturato** [comment]
Sorry I'm not able to do a screenshot now. I'm using numix theme. I think
at this point that the issue is not an issue, or better: the way gnome
change the battery icon is not continuous, it has few states. Maybe?
Il 23/Dic/2015 23:37, "dirtydancing" <notifications@github.com> ha scritto:

**Comment by dirtydancing** [comment]
@GabVenturato When you have a look at the two rows of Numix and Adwaita battery status icons above, is there any difference between the Numix and the Adwaita icons that appears on your screen?

**Comment by GabVenturato** [comment]
I'm not sure about  it, I will check it better tomorrow. Thank you and
sorry if I create useless confusion
Il 23/Dic/2015 23:43, "Andreas Angerer" <notifications@github.com> ha
scritto:

**Comment by dirtydancing** [comment]
No worries :-), the battery icons have caused quite some issues in the past, so it is always good to keep an eye on these. In case of doubt, always report. I have not noticed any problems with Arch/GNOME.

**Comment by GabVenturato** [comment]
I think that dirtydancing was right. I checked my battery indicator in a full discharging cycle, and I made some screenshots:

![screenshot from 2015-12-24 12-59-50](https://cloud.githubusercontent.com/assets/7904029/11996510/b8e99090-aa64-11e5-94fa-438b46aa711a.png)

![screenshot from 2015-12-24 12-53-31](https://cloud.githubusercontent.com/assets/7904029/11996513/c09d7ac2-aa64-11e5-8a3a-bb2ab97913a7.png)

![screenshot from 2015-12-24 14-22-19](https://cloud.githubusercontent.com/assets/7904029/11996521/cb0bc5ea-aa64-11e5-9802-8c64f32c397c.png)

![screenshot from 2015-12-24 14-39-34](https://cloud.githubusercontent.com/assets/7904029/11996522/ce878dbc-aa64-11e5-8506-a31512545450.png)

I can confirm that is not a numix issue, but it's apparently how GNOME works. I find this way to show battery status a lot confusing. 60% it's about half battery, and it indicates that it's full. I didn't know battery indicator thresholds, and sometimes checking the battery status I thought that there was a discrepancy between the icon shown and the real battery status.
Anyway thank you all for your support and sorry for the false allarmism that I created. 
And uhm, well, happy and merry christmas! :)
